tmk_core/common/print.h

```diff
@@ -31,6 +31,8 @@
 
 #if defined(PROTOCOL_CHIBIOS)
 #define PSTR(x) x
+#elif defined(PROTOCOL_ARM_ATSAM)
+#define PSTR(x) x
 #endif
```

Comment:
I thought to leave these separated since neither rely on the other. If a change is required in the future, they will need to be split out anyways or possibly break the other.

Reply:
The changes would have to be done manually anyways, so I don't think it's that big of a deal. If you feel strongly about keeping them separated, I'm okay with that.

Reply:
Opted for your suggestion